install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -130,7 +130,7 @@ data:
     outboundTrafficPolicy:
       mode: {{ .Values.global.outboundTrafficPolicy.mode }}
 
-    {{- if  .Values.global.localityLbSetting }}
+    {{- if  .Values.global.localityLbSetting.enabled }}
     localityLbSetting:
 {{ toYaml .Values.global.localityLbSetting | indent 6 }}
```

Comment:
I found the `LocalityLoadBalancerSetting` does not have the `enabled` field， will the unmarshal to yaml fail with `enabled` or unknown field is ignored currently?

Reply:
yes I've tested this. j think you added this feature